### PR TITLE
git-alias, git-clone, git-config: add pt_BR translation

### PR DESCRIPTION
**Pull request by caduvieira**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.pt_BR/common/git-clone.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# git clone
+
+> Clona um repositório exitente.
```
Comment (isaacvicente) [inline pages.pt_BR/common/git-clone.md:3]:
```suggestion
> Clona um repositório existente.
```

File: pages.pt_BR/common/git-config.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+> Gerencia configurações customizadadas para repositórios Git.
+> Estas configurações podem ser locais (para o repositório corrente) ou globais (para o usuário atual).
+> Mais informações: <https://git-scm.com/docs/git-config>.
+
+- Lista somente configurações locais (armazenadas no `.git/config` do repositório corrente):
+
+`git config --list --local`
+
+- Lista somente configurações globais (armazendas no `~/.gitconfig` por padrão ou no `$XDG_CONFIG_HOME/git/config` se tal arquivo existe):
```
Comment (isaacvicente) [inline pages.pt_BR/common/git-config.md:11]:
```suggestion
- Lista somente configurações globais (armazenadas no `~/.gitconfig` por padrão ou no `$XDG_CONFIG_HOME/git/config` se tal arquivo existe):
```

**Review by isaacvicente — COMMENTED** [review]
Corrigi alguns erros de digitação. Muito obrigado pela sua contribuição :100: :tada:.
Espero que contribua com o projeto mais vezes :)

**Comment by caduvieira** [comment]
Obrigado @isaacvicente !

**Review by isaacvicente — APPROVED** [review]
LGTM :tada:

**Pull request merged**